src/Sarif.Sarifer/Properties/VersionConstants.cs

```diff
@@ -1,13 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-namespace Microsoft.CodeAnalysis.Sarif.Sarifer
-{
-    public static class VersionConstants
-    {
-        public const string Prerelease = "";
-        public const string AssemblyVersion = "2.1.20";
-        public const string FileVersion = AssemblyVersion + ".0";
-        public const string Version = AssemblyVersion + Prerelease;
-    }
-}
```

Comment:
I don't see you deleting VersionConstants.cs from the Sarifer project.

Reply:
looking here, it's deleted. sarif.sarifer, sarif.viewer.visualstudio and sariv.viewer.visualstudio.interop

---
In reply to: [521585988](https://github.com/microsoft/sarif-visualstudio-extension/pull/267#discussion_r521585988) [](ancestors = 521585988)

Reply:
I see what happened. I never committed that file in that project. I didn't notice since it's regenerated on every build. Arguably I should never have committed them in any project in the first place.

---
In reply to: [521587263](https://github.com/microsoft/sarif-visualstudio-extension/pull/267#discussion_r521587263) [](ancestors = 521587263,521585988)